ocs_ci/framework/main.py

```diff
@@ -129,13 +129,12 @@ def process_ocsci_conf(arguments):
         action="store",
         choices=[
             "4.99",
-            "4.10",
-            "4.11",
```

Comment:
need 4.11 as  for testing upgrade scenarios for 4.12. we need to keep yaml files for 4.11 as well

Reply:
we don't run upgrade from 4.10 or 4.11 from master or stable so master/stable branch doesn't need to support n-5 or n-6, we support only n-1 in ocs-ci for now. So for now at least removing some non supported versions from master